modules/social_features/social_post/modules/social_post_album/templates/album-post-popup.html.twig

```diff
@@ -0,0 +1,26 @@
+{{ attach_library('core/drupal.dialog.ajax') }}
+{{ attach_library('social_post_album/social-post-album--popup') }}
+
+<div class="row social-post-album--popup">
+  <div class="social-post-album--popup-slider">
+    <div class="social-post-album--popup-slider-wrapper">
+      {% for url in urls %}
+        <div class="post-index-{{ index }} social-post-album--popup-item"
+           data-med="{{ url }}">
```

Comment:
Is this parameter needed when the PhotoSwipe library doesn't use here?

Reply:
Removed

Reply:
I meant the "data-med" attribute which still present.